db/migrate/20171103212000_add_active_users_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddActiveUsersToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :active, :boolean, default: false, null: false
```

Comment:
Maybe worth adding a `comment` to explain what `active` means?

Reply:
you mean a DB comment?

Reply:
Yes - only really relevant for development, but a QOL feature I've recently found valuable.

Reply:
Ahhh https://blog.bigbinary.com/2016/06/21/rails-5-supports-adding-comments-migrations.html